cmis/src/main/java/nl/nn/adapterframework/extensions/cmis/server/impl/IbisObjectService.java

```diff
@@ -136,7 +140,9 @@ public String createFolder(String repositoryId, Properties properties,
 			}
 			cmisXml.addSubElement(propertiesXml);
 
-			Element result = eventDispatcher.trigger(CmisEvent.CREATE_FOLDER, cmisXml.toXML());
+			IPipeLineSession context = new PipeLineSessionBase();
+			context.put(CmisUtils.CMIS_CALLCONTEXT_KEY, callContext);
+			Element result = eventDispatcher.trigger(CmisEvent.CREATE_FOLDER, cmisXml.toXML(), context);
```

Comment:
Deze drie regels zie ik wel heel vaak terugkomen. Zouden die gecombineerd kunnen worden?

Reply:
Ben bang van niet :( voor het aanroepen van de eventDispatcher moet de `callContext` gezet zijn in de `PipeLineSession`